source/extensions/filters/http/jwt_authn/jwks_cache.h

```diff
@@ -33,9 +36,9 @@ using JwksCachePtr = std::unique_ptr<JwksCache>;
  *     verifyJwt(jwks_data->getJwksObj(), jwt);
  */
 
-class JwksCache {
+class Cache {
```

Comment:
I decided not to rename this class.  Not to make too big change.

Reply:
Undo changes.